bin/sidecli.re

```diff
@@ -45,6 +59,8 @@ let exits =
 
 let lwt_ret = p => Term.(ret(const(Lwt_main.run) $ p));
 
+let const_log = p => Term.(const(() => p) $ setup_log);
```

Comment:
Can you explain the purpose of this function? It looks like it runs `setup_log`. Why do we have to run `setup_log` more than once?

Reply:
It should be only getting called once per sidecli invokation. If it's being called any more than that I've made a mistake